### PR TITLE
Added pause and unpause support

### PR DESCRIPTION
**Pull request by rustjson**

### CODE REVIEW

File: controller/static/app/containers/containers.controller.js [hunk 1]
```diff
@@ -227,6 +254,8 @@
 
         function containerStatusText(container) {
             if(container.Status.indexOf("Up")==0){
+                if (container.Status.indexOf("(Paused)") != -1)
```
Comment (tombee) [inline controller/static/app/containers/containers.controller.js]: Style nit:
Could we get braces around `return "Paused";`? :)

**Comment by tombee** [comment]
Thanks for this @RustJason!

Would you be able to add the equivalent action to the inspect page too, just to be consistent?

**Comment by rustjson** [comment]
I found a inconstancy about containers color, in containers page, stoped container display as gray while running container display green. But in inspect page, red shows as stoped container. am I right?

File: controller/static/app/containers/containers.controller.js (hunk 1 above)
Comment (rustjson) [inline controller/static/app/containers/containers.controller.js]: Ok. But another question ,I found you are using `container.state.Runing`  to do this branch checking ,can we make this for constancy?

**Comment by tombee** [comment]
Good catch -- yup, let's make these consistent.  Maybe we can change these to traffic light colours? 

:traffic_light: 
Green - Running
Amber/Orange - Paused
Red - Stopped

**Comment by ehazlett** [comment]
Nice! :+1:

**Comment by rustjson** [comment]
:smile: OK

**Comment by RustJason** [comment]
@tombee Done with that, any other suggestions?

**Comment by rustjson** [comment]
@tombee Done with that, any other suggestions?

**Comment by tombee** [comment]
@RustJason Awesome, thanks for this.  Looks good.. I'll get around to testing it a little later :)

**Comment by rustjson** [comment]
LOL, seeing 'looks good' rather then LGTM makes me feel not good enough. hahahha :laughing:

**Comment by tombee** [comment]
Hah, give me a chance to test it out first! ;)

**Comment by rustjson** [comment]
@tombee  I want to code more to this project, I can see you and @ehazlett are the main maintainers, How do you cooperate and can I participate in with other function? thx

**Comment by tombee** [comment]
The #shipyard channel on IRC Freenode is the best place to come get
involved and discuss ideas.

**Comment by tombee** [comment]
Nice work, just tested it out.  

Sorry I missed your message on IRC btw, call by again and ping @ehazlett and I in #shipyard :)

LGTM!

**Pull request merged**